go.mod

```diff
@@ -17,11 +17,13 @@ require (
 	github.com/elastic/elastic-agent-shipper-client v0.4.0
```

Comment:
The elastic-agent-client dependency above this one needs to be updated to v7.0.3: https://github.com/elastic/elastic-agent-client